addons/account/wizard/account_move_send.py

```diff
@@ -177,10 +175,7 @@ def _compute_company_id(self):
     @api.depends('move_ids')
     def _compute_mode(self):
         for wizard in self:
-            mode = 'done'
-            if wizard.move_ids and all(x.is_invoice(include_receipts=True) for x in wizard.move_ids):
-                mode = 'invoice_single' if len(wizard.move_ids) == 1 else 'invoice_multi'
-            wizard.mode = mode
+            wizard.mode = 'invoice_single' if len(wizard.move_ids) == 1 else 'invoice_multi'
```

Comment:
It could be something else than an invoice. For now we don't have an example but we will need it in the future

Reply:
Then let's add it back when necessary ? Else what would be your default value when not an invoice ? :face_with_spiral_eyes: